host/windows/dynload.c

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include "../dynload.h"
+#include <Windows.h>
+
+void* oe_shared_library_load(const char* name)
+{
+    return (void*)LoadLibraryEx(name, NULL, 0);
```

Comment:
Consider using the `LOAD_LIBRARY_SEARCH_SYSTEM32` flag since sgx_enclave_common.dll is part of the Intel driver components and should only be loaded from there.

Reply:
Addressed in #3155